### DISCUSSION TITLE
Looking for practice with numpy and large datasets

### DISCUSSION THREAD

**Discussion by FinShot** [community/community#126295] [Programming Help]
### Select Topic Area

Question

### Body

I've been spending a lot of time learning Numpy, but now I would like to practice as much as possible. 

Recently I worked with a csv file containing about 8000+ rows and 20 columns on taxi data from NYC.  I calculated the mean speed, the number of rides in February, number of dropoffs at JFK airport... it was PERFECT. 

I'm new to Github... can someone help point me in the right direction? My searches are turning up blank. 

Thank you!

**Comment by davevad93** [comment]
Hi @FinShot ,

I think the best place for datasets is [Kaggle](https://www.kaggle.com/), however here are some GitHub repos that you might find interesting:

1. [awesome-public-datasets](https://github.com/awesomedata/awesome-public-datasets)
2. [datasets](https://github.com/datasets)
3. [A bunch of some 200 datasets](https://github.com/MainakRepositor/Datasets)
4. [huggingface datasets](https://github.com/huggingface/datasets)
5. [tensorflow datasets](https://github.com/tensorflow/datasets)

To search for more datasets in GitHub see [here](https://github.com/search?q=datasets&type=repositories).

**Reply by FinShot** [reply]
Wow thank you!

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @FinShot ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by FinShot** [reply]
Thanks for your help! This was my first ever post in GH.
